circle.yml

```diff
@@ -7,13 +9,21 @@ machine:
     - cf -v
 
 dependencies:
+  pre:
+  - make pa11y-install
```

Comment:
Does this need to be in `pre` or can it be appended/prepended to `override`? It doesn't really matter but semantically it looks like it's setting up a dependency for the actual tests rather than something that has to happen before the other dependencies are installed.